Add basic structure 2

Aichnerc opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no FAQ section. #16 

**What is the new behavior (if this is a feature change)?**
- The basic structure of the FAQ section has been added. #16

Aichnerc committed:
Add vscode settings

The settings of visual studio code has been
added.

Aichnerc committed:
Add Section FAQ

The section FAQ has been added without any
content.

Aichnerc committed:
Add Section FAQ structure

The basic structure of the FAQ section has been
added.

Aichnerc committed:
Improve code quality

The code quality has been improved due to a
notice by Codacy.
Msg: Missing semicolon. (semi)

kleberbaum commented on src/components/organisms/Sections/FAQ/index.jsx:
```diff
@@ -0,0 +1,61 @@
+    },
+    {
+        icon: "leaf",
+        heading: "Schmeckt dieser Kaffee gleich?",
```
This doesn't sound like a proper German question. xD

Aichnerc replied:
Content is not the objective of this pull request.

kleberbaum reviewed (changes requested):
Should be ok to merge after a few changes.

Aichnerc committed:
Improve content

An issue unrelated change in the grammar of a
placeholder text has been added due to a 
request of @kleberbaum.

kleberbaum reviewed (approved):
Should be ok to merge.
